react/package.json

```diff
@@ -11,8 +11,10 @@
     "@apollo/react-testing": "^3.1.4",
     "@vtex/test-tools": "^3.1.0",
     "vtex.css-handles": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.css-handles@1.0.0/public/@types/vtex.css-handles",
-    "vtex.product-context": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.product-context@0.9.7/public/@types/vtex.product-context",
-    "vtex.render-runtime": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.render-runtime@8.126.11/public/@types/vtex.render-runtime"
+    "vtex.native-types": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.native-types@0.7.5/public/@types/vtex.native-types",
+    "vtex.product-availability": "https://victormiranda--storecomponents.myvtex.com/_v/private/typings/linked/v1/vtex.product-availability@0.2.0+build1645631307/public/@types/vtex.product-availability",
```

Comment:
What is this?

Reply:
This is the typescript types for the app :)

Reply:
But I don't think the app should be including the types for itself 🤔 .
The types that I wanted to bring here were the ones from native-types

Reply:
Yeah, it does. We talked about it some time ago. It happens for the cases when the app has a graphql and use types' from itself